### DISCUSSION TITLE
add to documentation: GPU on Docker Desktop - only for WSL

### DISCUSSION THREAD

**Discussion by cunicode** [immich-app/immich#14404] [Q&A]
After many unsuccessful attempts to enable GPU in my linux setup, I realised that GPU support on Docker Desktop is only for WSL.

[Currently GPU support in Docker Desktop is only available on Windows with the WSL2 backend. ](https://docs.docker.com/desktop/features/gpu/)

[Shall we add a note to the docs?](https://immich.app/docs/features/ml-hardware-acceleration/)

**Comment by mmomjian** [comment]
Our docs cover this: "Only Linux and Windows (through WSL2) servers are supported"

**Comment by mertalev** [comment]
I think there are some nuances like Docker on macOS and Hyper-V vs WSL2 on Windows that could use more clarification. The line about only Linux and Windows being supported is also kinda misleading since it works on macOS too.

**Comment by mertalev** [comment]
Oh and for reference, GPU support is totally there for Linux. You're misreading the note in the Docker article: *on Windows*, GPU support requires WSL2.

**Comment by cunicode** [comment]
I feel the sentence is misleading, or I'm reading it wrong

what I read here is:
_Only Linux and Windows (through WSL2) servers are supported_ 
- on linux = OK
- Windows (through WSL2) servers are supported = OK

but my experience is:
- in linux via docker engine = OK it works 
- in linux via docker desktop = not work (as per docker docs: "[Currently GPU support in Docker Desktop is only available on Windows with the WSL2 backend.](https://docs.docker.com/desktop/features/gpu/)"

I guess my question is: does anyone managed to make immich GPU work in docker desktop linux?

**Reply by bo0tzz** [reply]
There's no reason to use docker desktop on linux
